frontend/src/shared/helpers.tsx

```diff
@@ -1041,9 +1045,13 @@ export function prepareSampleCohortDataForAgGrid(samples: Sample[]) {
       });
     });
     const deliveryDate = cohortDates?.sort()[0]; // earliest cohort date
-    let embargoDateAsDate = new Date(deliveryDate);
-    embargoDateAsDate.setMonth(embargoDateAsDate.getMonth() + 18); // embargo date is 18 months post earliest delivery date
-    const embargoDate = moment(embargoDateAsDate).format("YYYY-MM-DD");
+
+    let embargoDate;
+    if (deliveryDate !== undefined) {
+      let embargoDateAsDate = new Date(deliveryDate);
+      embargoDateAsDate.setMonth(embargoDateAsDate.getMonth() + 18);
+      embargoDate = moment(embargoDateAsDate).format("YYYY-MM-DD");
+    }
```

Comment:
In the Cohort Sample view, when a sample does not have any Tempo events:

**Before**
![before](https://github.com/mskcc/smile-dashboard/assets/86090707/fa4d4655-1acd-45a7-bf11-ee457dcf7274)
"Invalid date" string is set by `moment(embargoDateAsDate)` due to `embargoDateAsDate` being undefined

**After**
![after](https://github.com/mskcc/smile-dashboard/assets/86090707/3a68351a-dac4-4ce9-b29f-d5cbb0434750)